flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -76,6 +76,39 @@ public class CheckpointingOptions {
      * CheckpointStorageFactory#createFromConfig(ReadableConfig, ClassLoader)} method is called.
      *
      * <p>Recognized shortcut names are 'jobmanager' and 'filesystem'.
+     *
+     * <p>{@link #CHECKPOINT_STORAGE} and {@link #CHECKPOINTS_DIRECTORY} are usually combined to
+     * configure the checkpoint location. The behaviors of different combinations are as follows:
```

Comment:
Instead of combining these two options, I'd suggest introduce the behaviors when set CHECKPOINT_STORAGE='jobmanager' and CHECKPOINT_STORAGE='filesystem' seperately. WDYT?

Reply:
👍Thanks for the suggestion, I reorganized the description as you suggested.